docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04 as web-base
+
+RUN apt update && apt install -y git curl
```

Comment:
`RUN apt-get update && apt-get upgrade && apt-get install -y git curl`
That is, shouldn't there be an "upgrade" after the "update"?

Reply:
yeah... it doesn't hurt.

Reply:
yeah... it doesn't hurt.